src/components/plots/plot.ts

```diff
@@ -540,5 +563,44 @@ module Plottable {
       return !(pixelPoint.x < 0 || pixelPoint.y < 0 ||
         pixelPoint.x > this.width() || pixelPoint.y > this.height());
     }
+
+    protected _updateExtentsForProperty(property: string) {
+      var accScaleBinding = this._propertyBindings.get(property);
+      if (accScaleBinding.accessor == null) { return; }
```

Comment:
This is worthwhile since there could be accessors that are not necessarily defined before this is called.

I believe for example this is called in `addDataset` but the property may not be set before that call